.changeset/red-trains-help.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sigstore/sign': major
+---
+
+Bump `make-fetch-happen` from 11.0.0 to 12.0.0
```

Comment:
Technically, `make-fetch-happen` was also updated in the `sigstore` package -- however, I'm going to remove this dependency altogether in a subsequent PR so I won't bother including this note in the package's changelog.